src/main/java/uk/gov/companieshouse/filetransferservice/controller/FileTransferController.java

```diff
@@ -136,8 +136,8 @@ public ResponseEntity<FileApi> downloadAsJson(@PathVariable String fileId) throw
      * @param fileId of remote file
      * @return file data
      */
-    @GetMapping(path = "/{fileId}/download")
-    public ResponseEntity<byte[]> download(@PathVariable String fileId, @RequestParam(defaultValue = "false") boolean bypassAv) throws FileNotFoundException, FileNotCleanException {
+    @GetMapping(path = "/{fileId}/downloadbinary ")
```

Comment:
There might be an accidental trailing space here

```suggestion
    @GetMapping(path = "/{fileId}/downloadbinary")
```